README.md

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Delete unnecessary new line at the begining

Reply:
Done